src/features/common/Accordion.tsx

```diff
@@ -0,0 +1,53 @@
+import * as Accordion from "@radix-ui/react-accordion";
+import { MdChevronRight } from "react-icons/md";
+import { twJoin } from "tailwind-merge";
+import { ReactNode, useState } from "react";
+
+interface AccordionComponentProps {
+  rootStyles: string;
+  type?: "single" | "multiple";
+  itemId: string;
+  trigger: ReactNode;
+  content: ReactNode;
+}
+
+const AccordionComponent = ({
+  rootStyles,
+  type = "multiple",
+  itemId,
+  trigger,
+  content,
+}: AccordionComponentProps) => {
+  const [isAccordionOpen, setIsAccordionOpen] = useState<boolean>(false);
+  const toggleChevronIconUp = () =>
```

Comment:
minor but I'd call this `toggleAccordion` - as is it sounds like it's toggling the icon.